### PR TITLE
Add Mute Button

### LINKED ISSUE #1: Add general mute button

### PR DESCRIPTION
**Pull request by jacklu97**
Closes #1 

This PR enables a general toggle button that mutes/unmutes the whole system.
Small changes to the UI were performed:

- New section that includes the mute toggler
- New sub titles to represent the information displayed on the sections, this to add more readability to the plugin
- Now all the applications that use VolumeControl component display the volume level with a number

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
+import GeneralSoundToggle from "../GeneralSoundToggle/GeneralSoundToggle";
 
 
 const App: VFC = () => {
   return (
-    <PanelSection>
-      <div className={staticClasses.PanelSectionTitle}>
-        {/* Music Mixer */}
-      </div>
+    <Fragment>
```
Comment (raul-cayo) [inline src/components/App/App.tsx]: You could use the [Fragment shorthand](https://thisthat.dev/react-fragment-shorthand-vs-longhand) `<></>` like the cool kids 😎

File: src/services/pythonServer.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ class PythonServer {
     return this.server
   }
 
-  resolve(promise: Promise<any>, setter: any) {
+  resolve(promise: Promise<any>, setter?: any) {
```
Comment (raul-cayo) [inline src/services/pythonServer.ts:25]: Is the `setter` really optional? Seems like you always execute it on `data.success`

Comment (jacklu97) [reply src/services/pythonServer.ts:25]: The idea was that sometimes it is not required to get any feedback from the server as the server side commands don't return any useful data. However, later on I decided it was better to actually implement some return statuses. 
This won't be optional on a future PR

File: src/components/App/App.tsx (hunk 1 above)
Comment (jacklu97) [reply src/components/App/App.tsx]: Compiler config updated, shorthand used instead

**Review by raul-cayo — APPROVED** [review]

**Pull request merged**